datahub-gms-graphql-service/README.md

```diff
@@ -427,3 +427,142 @@ Sample Response:
   }
 }
 ```
+
+### Query DataFlow
+
+Request:
+
+```
+{
+  dataFlow(urn: "urn:li:dataFlow:(airflow,flow1,foo)") {
+    urn
+    type
+    orchestrator
+    flowId
+    info {
+      name
+      description
+      project
+    }
+    ownership {
+      owners {
+        owner {
+          username
+          urn
+          info {
+            displayName
+            email
+            fullName
+            manager {
+              urn
+            }
+          }
+          editableInfo {
+            aboutMe
+            skills
+          }
+        }
+        type
+        source {
+          url
+        }
+      }
+      lastModified {
+        actor
+      }
+    }
+	}
+}
+```
+
+Sample response:
+
+```
+{
+  "data": {
+    "dataFlow": {
+      "urn": "urn:li:dataFlow:(airflow,flow1,foo)",
+      "type": "DATA_FLOW",
+      "orchestrator": "airflow",
+      "flowId": "flow1",
+      "info": {
+        "name": "flow1",
+        "description": "My own workflow",
+        "project": "X"
+      },
+      "ownership": {
+        "owners": [
+          {
+            "owner": {
+              "username": "test-user",
+              "urn": "urn:li:corpuser:test-user",
+              "info": null,
+              "editableInfo": null
+            },
+            "type": "DEVELOPER",
+            "source": null
+          }
+        ],
+        "lastModified": {
+          "actor": "urn:li:corpuser:datahub"
+        }
+      }
+    }
+  }
+}
+```
+
+### Query DataJob
+
+Request:
+
+```
+{
+  dataJob(urn: "urn:li:dataJob:(urn:li:dataFlow:(airflow,flow1,foo),task1)") {
+    urn
+    type
+    jobId
+    dataFlow {
+      urn
```

Comment:
Hopefully fetching all the fields of dataFlow will work :)

Reply:
yes, I just kept example short